content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+* Lihat halaman [Cluster Networking](/docs/concepts/cluster-administration/networking/) untuk daftar
+_add-on_ jaringan Pod yang lebih banyak.
+* <a id="other-addons" />Lihat [daftar _add-on_](/docs/concepts/cluster-administration/addons/) untuk
+  mengeksplor _add-on_ lainnya, termasuk perangkat untuk _logging_, _monitoring_, network policy, visualisasi &amp;
+  pengendalian klaster Kubernetes.
+* Atur bagaimana klaster mengelola _log_ untuk peristiwa-peristiwa klaster dan dari
+  aplikasi-aplikasi yang berjalan pada Pod.
+  Lihat [Arsitektur Logging](/docs/concepts/cluster-administration/logging/) untuk
+  gambaran umum tentang hal-hal yang terlibat.
+
+### Umpan balik {#feedback}
+
+* Untuk _bug_, kunjungi [kubeadm GitHub issue tracker](https://github.com/kubernetes/kubeadm/issues)
+* Untuk dukungan, kunjungi kanal Slack
+  [#kubeadm](https://kubernetes.slack.com/messages/kubeadm/)
+* Kanal Slack umum pengembangan SIG Cluster Lifecycle:
+  [#sig-cluster-lifecycle](https://kubernetes.slack.com/messages/sig-cluster-lifecycle/)
+* SIG Cluster Lifecycle [SIG information](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle#readme)
+* Milis SIG Cluster Lifecycle:
+  [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+
+## Kebijakan versi _skew_ {#version-skew-policy}
+
+`kubeadm` versi v{{< skew latestVersion >}} dapat menyebarkan klaster dengan _control plane_ versi v{{< skew latestVersion >}} atau v{{< skew prevMinorVersion >}}.
+`kubeadm` v{{< skew latestVersion >}} juga dapat memperbarui klaster yang dibuat dengan kubeadm v{{< skew prevMinorVersion >}}.
+
+Karena kita tidak dapat melihat masa depan, kubeadm CLI v{{< skew latestVersion >}} mungkin atau tidak mungkin dapat menyebarkan klaster v{{< skew nextMinorVersion >}}.
+
+Sumber daya ini menyediakan informasi lebih lanjut mengenai versi _skew_ yang didukung antara kubelet dan _control plane_, serta komponen Kubernetes lainnya:
+
+* Kubernetes [version and version-skew policy](/docs/setup/release/version-skew-policy/)
+* Kubeadm-specific [panduan instalasi](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl)
+
+## Keterbatasan {#keterbatasan}
+
+### Ketahanan klaster {#ketahanan}
+
+Klaster yang dibuat pada panduan ini hanya memiliki simpul _control-plane_ tunggal, dengan basis data etcd tunggal
+yang berjalan di atasnya. Hal ini berarti jika terjadi kegagalan pada simpul _control-plane_, klaster dapat kehilangan
+data dan mungkin harus dibuat kembali dari awal.
+
+Solusi:
+
+* Lakukan [back up etcd](https://coreos.com/etcd/docs/latest/admin_guide.html) secara reguler. Direktori data
+  etcd yang dikonfigurasi oleh kubeadm berada di `/var/lib/etcd` pada simpul _control-plane_.
+
+* Gunakan banyak simpul _control-plane_. Kamu dapat membaca
```

Comment:
```suggestion
* Gunakan banyak Node _control-plane_. Kamu dapat membaca
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc